### PR TITLE
Egfad 260

### PR DESCRIPTION
**Pull request by duncanmeech**

### CODE REVIEW

File: src/containers/GlobalNav.js [hunk 1]
```diff
@@ -26,10 +26,17 @@ import {
...
  } from '../selectors/blocks';
 import { projectGetVersion } from '../selectors/projects';
 import { undo, redo } from '../store/undo/actions';
-import { uiShowGenBankImport } from '../actions/ui';
+import {
+  uiShowGenBankImport,
+  uiToggleDetailView,
+  uiSetGrunt,
+ } from '../actions/ui';
+import { inspectorToggleVisibility } from '..//actions/inspector';
```
Comment (maxbates) [inline src/containers/GlobalNav.js]: wrong path

File: src/containers/GlobalNav.js [hunk 2]
```diff
@@ -413,6 +416,8 @@ function mapStateToProps(state) {
     focus: state.focus,
     blocks: state.blocks,
     clipboard: state.clipboard,
+    inspector: state.inspector.isVisible,
```
Comment (maxbates) [inline src/containers/GlobalNav.js]: can you more clearly name

File: src/containers/GlobalNav.js [hunk 3]
```diff
@@ -413,6 +416,8 @@ function mapStateToProps(state) {
...
+    inventory: state.inventory.isVisible,
```
Comment (maxbates) [inline src/containers/GlobalNav.js]: name

Comment (duncanmeech) [inline src/containers/GlobalNav.js]: done

**Pull request merged**